docs/Community/CONTRIBUTING.md

```diff
@@ -1,16 +1,213 @@
+  Als u zich slechts op een minderheid van gebruikers richt, overweeg dan om een bibliotheek met add-ons/plug-ins te
+  schrijven.
+
+### Hoe dien ik een goede verbeteringssuggestie in?
+
+Suggesties voor verbeteringen worden bijgehouden als [GitHub issues](https://github.com/ospo-nl/kennisbank/issues).
+
+- Gebruik een **duidelijke en beschrijvende titel** voor het probleem om de suggestie te identificeren.
+- Geef een stapsgewijze beschrijving van de voorgestelde verbetering met zoveel mogelijk details.
+- Beschrijf het huidige gedrag en leg uit welk gedrag je in plaats daarvan verwachtte te zien en waarom. Op dit punt
+  kunt u ook zien welke alternatieven niet voor u werken.
+- Misschien wilt u schermafbeeldingen en geanimeerde GIF's toevoegen die u helpen de stappen te demonstreren of aan te
+  geven op welk onderdeel de suggestie betrekking heeft. U kunt deze tool gebruiken om GIF's op macOS en Windows op te
+  nemen, en deze tool of deze tool op Linux.
+- Leg uit waarom deze verbetering nuttig zou zijn voor de meeste gebruikers van OSPO-NL. Misschien wil je ook wijzen op
+  de andere projecten die het beter hebben opgelost en die als inspiratie kunnen dienen.
+
+# Review proces
+
+Om wijzigingen goed te kunnen beheren, volgen en uit te leggen, volgen we een eenvoudig proces van review en Pull
+Requests (PRs).
+
+- Wijzigingen wordt nooit direct op de `main` branch gedaan, maar altijd in een 'feature' branch.
+- Maak een Pull Request aan zodra je klaar bent. Of maak gelijk een Draft Pull Request aan nadat u uw feature branch
+  hebt aangemaakt. Zodra uw wijzigingen klaar zijn voor review, wijzigt u uw Draft PR naar 'Ready for Review'.
+- Een teamlid reviewt de wijzigingen in het Pull Request en geeft commentaar en/of goedkeuring (Approve).
+- Na goedkeuring kan het Pull Request gemerged worden. Hiervoor wordt standaard 'Squash & Merge' gebruikt. In geval dat
+  de PR door een teamlid is ingediend, reviewt een ander teamlid de PR maar wordt de merge overgelaten aan de auteur van
+  de PR.
+- Na de merge dienen bijbehorende issues bijgewerkt te worden zodat deze niet onnodig open blijven staan ofwel
+  beantwoord worden.
+
+# Styleguides
+
+## Markdown
+
+Alle documentatie moet 'machine-readable' zijn en tegelijk ook makkelijk leesbaar en onderhoudbaar voor mensen. Daarom
+maken we gebruik van **Markdown**. Zie ook [GitHub
+Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
+en de algemene [Markdown handleiding](https://www.markdownguide.org/basic-syntax/) (EN) (of zelfs de originele
+[documentatie](https://daringfireball.net/projects/markdown/syntax)).
+
+Alinea's worden op of binnen **120 karakters** afgekapt om versiebeheer per regel beheersbaar te maken. Dit kan
+automatisch worden afgedwongen in tooling, bijv. [Rewrap in VSCode](https://stkb.github.io/Rewrap/) (keyboard shortcut:

```

Comment:
Zelf hanteer ik het principe 1 zin per regel met hetzelfde doel maar andere uitwerking. Beiden acceptabel wat mij betreft, zolang regels niet te lang worden.

Reply:
Ja, ik wissel die nog wel eens ... maar het helpt als we iets kiezen en alles daar aan voldoet. Aantal karakters is gemakkelijk mbv tools af te dwingen ... en ik denk dat ik dat net iets vaker tegenkom in andere repos ... dus is dat de de facto standaard? 🤔